### DISCUSSION TITLE
Cramped book list

### DISCUSSION THREAD

**Discussion by Original-Runsha** [audiamus/BookLibConnect#132] [Q&A]
It is only a minor annoyance, but as my eyes get older the cramped list spacing gets harder and harder to read. Is there any way to adjust the cell spacing to match the letter font size?

![CrampedList](https://github.com/audiamus/BookLibConnect/assets/119818852/beba4836-9af2-43d2-b948-d56f41f90b85)

**Comment by audiamus** [comment]
That looks like something being wrong with your system settings. I don't think it has anything to do with Book Lib Connect directly. 

What you normally can do in Windows 10 and 11 is to set your display font size to 125% or 150%, under (Windows) Settings, Display, Scale and Layout. This will affect all apps, of course. Book Lib Connect should show all controls with the larger font then, hereby automatically arranging the windows to the slightly different aspect ratio of the larger fonts.

**Comment by Original-Runsha** [comment]
Actually already running with 300% which Windows says is recommended. Just
for grins I reset it to 100, 125, 150, 200, & 250, but only thing it seems
to effect is size of the window (proportionally) and the top half of the
display. The lower title selections remain larger than the framing for each
cell.

Not an earth shattering issue. Just seems to be an issue limited to Book
Lib Connect, so thought would ask.

Thanks for the wonderful pair of applications! And thanks for the response.
